SADL/AM-CDM-part.sadl

```diff
@@ -0,0 +1,31 @@
+uri "http://kdl.ge.com/AM-CDM-part" alias AM-CDM-part version "$Revision:$ Last modified on   $Date:$".
+
+import "http://kdl.ge.com/AM-CDM-base".
+
+Part (note "Instances of part designs in the as-built state.") is a MaterialStock,
+	described by name (note 'Human readable name which designates this part.') with a single value of type string,
+	described by processHistory with a single value of type ProcessHistory,
```

Comment:
Have we decided to eliminate partID?  I'm thinking an org may have a human-readable name for a part, and an alphanumerical identifier. This goes with the conversation we did not fully answer, "what needs an ID in the CDM"

Reply:
Exercise across all classes in all sadl files consistently calling out ID and human readable names should be revisited. Action for _all_